diagrams-svg.cabal

```diff
@@ -61,6 +62,7 @@ Library
                      , lens                 >= 4.0   && < 4.12
                      , hashable             >= 1.1   && < 1.3
                      , optparse-applicative >= 0.10  && < 0.12
+                     , semigroups
```

Comment:
That's a fair point. I guess that since none of the `diagrams` packages have a ton of direct dependencies, that probably doesn't create an overwhelmingly horrible amount of work for you. Do you have some type of automation that figures out when new versions of dependencies are released and alerts you, or is the process more manual?